http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/KubernetesAppConfig.scala

```diff
@@ -87,6 +87,19 @@ final case class CoaAppConfig(chartName: ChartName,
     .toSet
 }
 
+final case class WdsAppConfig(chartName: ChartName,
+                              chartVersion: ChartVersion,
+                              releaseNameSuffix: ReleaseNameSuffix,
+                              namespaceNameSuffix: NamespaceNameSuffix,
+                              ksaName: KsaName,
+                              services: List[ServiceConfig],
+                              instrumentationEnabled: Boolean
+) extends KubernetesAppConfig {
+  override lazy val kubernetesServices: List[KubernetesService] = services.map(s => KubernetesService(ServiceId(-1), s))
+  override val serviceAccountName = ServiceAccountName(ksaName.value)
+  def coaServices: Set[CoaService] = Set(Wds)
+}
+
 sealed trait CoaService
```

Comment:
Do we eventually need something similar here for `Wds` as a service?

Reply:
I don't think so. These are used as an enum to handle different cases for different Coa services, so Wds won't need this unless we are devising a system of conditionally deploying apps on WDS. Which we aren't now :)